src/pages/NotFound.test.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import Enzyme, { shallow } from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
+import NotFound from "./NotFound";
+
+Enzyme.configure({adapter: new Adapter()})
+
+describe("When NotFound renders", () => {
+    it("displays 404 page", () => {
```

Comment:
this test is checking for an "a" tag. May want to refactor the name of the it statement to match.